requirements/adjust_versions.py

```diff
@@ -5,7 +5,6 @@
 
 # IMPORTANT: this list needs to be sorted in reverse
 VERSIONS = [
-    dict(torch="1.10.0", torchvision="0.11.*", torchtext=""),  # nightly
```

Comment:
This should stay, has no impact if there are no jobs with this configuration and will be useful later